### PR TITLE
Added code-block wrappers

### LINKED ISSUE #79031: Docs: Add code-block wrappers to code examples in implicit_localhost.rst
### Summary

*Problem**:
Throughout the Ansible docs, there are instances where example code is preceded with a lead-in sentence ending in `::`.
Translation programs then attempt to translate this code, which we don't want.

**Solution:**
Enclose code  in a `.. code-block:: <lexer>` element, so that translation processes know to skip this content.
For a list of allowed values for _`<lexer>`_ , refer to [Syntax highlighting - Pygments](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#syntax-highlighting-pygments).

**Scope:**
In the `implicit_localhost.rst ` file in the `docs/docsite/rst/inventory/` directory, there are 2 instances of lead-in sentences ending with `::`. Use the following `grep` command to identify the files and line numbers:

```
$ grep -rn --include "*.rst" "^[[:blank:]]*[^[:blank:]\.\.].*::$" .
```
$ grep -rn --include "*.rst" "^[[:blank:]]*[^[:blank:]\.\.].*::$" . | grep developing_collections_distributing.rst
```

**Example:**

Before:
```
Before running ``ansible-playbook``, run the following command to enable logging::

    export ANSIBLE_LOG_PATH=~/ansible.log

```

After:
```
Before running ``ansible-playbook``, run the following command to enable logging:

.. code-block:: shell

    export ANSIBLE_LOG_PATH=~/ansible.log

```

This problem has been addressed in some other guides; view these merged PRs to help get you started:
- Network Guide: [#75850](https://github.com/ansible/ansible/pull/75850/files) 
- Developer Guide: [#75849](https://github.com/ansible/ansible/pull/75849/files) 


### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/inventory/implicit_localhost.rst

### Ansible Version

```console
n/a
```

### Configuration

```console
n/a
```

### OS / Environment

n/a

### Additional Information

When example code is enclosed within a code-block element, translation programs do not attempt to translate the code.

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by 12-malak**
##### SUMMARY
Fixed the docs to display code blocks better In the `implicit_localhost.rst` file

Fixes #79031 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/inventory/implicit_localhost.rst`

### CODE REVIEW

File: docs/docsite/rst/inventory/implicit_localhost.rst [hunk 1]
```diff
@@ -5,15 +5,19 @@
 Implicit 'localhost'
 ====================
 
-When you try to reference a ``localhost`` and you don't have it defined in inventory, Ansible will create an implicit one for you.::
+When you try to reference a ``localhost`` and you don't have it defined in inventory, Ansible will create an implicit one for you.:
+
+.. code-block:: text
```
Comment (Akasurde) [inline docs/docsite/rst/inventory/implicit_localhost.rst:10]:
```suggestion
.. code-block:: yaml
```

File: docs/docsite/rst/inventory/implicit_localhost.rst [hunk 2]
```diff
@@ -5,15 +5,19 @@
...
     - hosts: all
       tasks:
         - name: check that i have log file for all hosts on my local machine
           stat: path=/var/log/hosts/{{inventory_hostname}}.log
           delegate_to: localhost
 
-In a case like this (or ``local_action``) when Ansible needs to contact a 'localhost' but you did not supply one, we create one for you. This host is defined with specific connection variables equivalent to this in an inventory::
+In a case like this (or ``local_action``) when Ansible needs to contact a 'localhost' but you did not supply one, we create one for you. This host is defined with specific connection variables equivalent to this in an inventory:
+
+.. code-block:: text
```
Comment (Akasurde) [inline docs/docsite/rst/inventory/implicit_localhost.rst:20]:
```suggestion
.. code-block:: yaml
```

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by 12-malak** [comment]
@Akasurde New PR pushed #79086 . Done the requested changes.

**Comment by Akasurde** [comment]
Superseded by #79086

**Pull request closed without merging**